### DISCUSSION TITLE
Localized DatePicker control

### DISCUSSION THREAD

**Discussion by azhago** [flet-dev/flet#2672] [Ideas]
Hi,
Is it possible to add localized version of that class?
I see there is commented "locale" parameters in the class but i can't figured out how to use localized date (day, month etc) or to change date format 
(Month/day/year is a weird format for non-US users)

**Comment by RRozi** [comment]
Same question....

**Comment by ndonkoHenri** [comment]
https://python.plainenglish.io/app-localization-in-flet-5b523e83ca89

**Reply by sagittarius-H** [reply]
Hi @ndonkoHenri, as far as I understood from announcements, starting from Flet v0.22.0 was announced build-in localization "locale_configuration" for page and it's childs, build-in in Flet widgets. Could you please clarify if localization for custom widgets is on roadmap?
